code/modules/deathmatch/deathmatch_mapping.dm

```diff
@@ -10,3 +10,15 @@
 
 /obj/effect/landmark/deathmatch_player_spawn
 	name = "Deathmatch Player Spawner"
+
+/datum/action/cooldown/spell/chuuni_invocations/deathmatch
+	name = "Unrestricted Chuuni Invocations"
+	spell_requirements = SPELL_REQUIRES_NO_ANTIMAGIC|SPELL_REQUIRES_MIND
+
+/datum/action/cooldown/spell/rod_form/deathmatch
+	name = "Unrestricted Rod Form"
+	spell_requirements = SPELL_REQUIRES_NO_ANTIMAGIC
+
+/datum/action/cooldown/spell/conjure/the_traps/deathmatch
+	name = "Unrestricted The Traps"
+	spell_requirements = SPELL_REQUIRES_NO_ANTIMAGIC
```

Comment:
These seem redundant, you could just remove these requirements when the spell datum is created
IE
```dm
	for(var/datum/action/act as anything in granted_spells)
		var/datum/action/new_ability = new act(user)
		if(istype(new_abililty, /datum/action/cooldown/spell)
		    var/datum/action/cooldown/spell/new_spell = new_ability
		    new_spell.spell_requirements &= ~SPELL_REQUIRES_STATION
		new_ability.Grant(user)
```

Reply:
addressed